### PR TITLE
workflows: add CI token

### PR DESCRIPTION
**Pull request by eivindj-nordic**
Add token for workflows using nordicbuilder/action-checkout-west-update.

**Secret used here should not exist in this repository; it has intentionally been left empty.**

### CODE REVIEW

**Comment by karhama** [comment]
secret used here should not exist in this repository; it has intentionally been left empty.

**Review by karhama — APPROVED** [review]

File: .github/workflows/docbuild.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ jobs:
     steps:
       - name: Checkout sources
         uses: nordicbuilder/action-checkout-west-update@main
+        with:
+          west-token: ${{ secrets.WEST_UPDATE_TOKEN }}
```
Comment (gmarull) [inline .github/workflows/docbuild.yml:36]: why is this needed here?

**Review by gmarull — CHANGES_REQUESTED** [review]
docs do not run west update

**Comment by eivindj-nordic** [comment]
> docs do not run west update

Where do you see that?

**Comment by carlescufi** [comment]
> > docs do not run west update
> 
> Where do you see that? To me the checkout sources step looks ok here: https://github.com/nrfconnect/sdk-nrf/actions/runs/8847144566/job/24294387285?pr=15105

@gmarull ping

**Comment by eivindj-nordic** [comment]
> > docs do not run west update
> 
> Where do you see that? To me the checkout sources step looks ok here: https://github.com/nrfconnect/sdk-nrf/actions/runs/8847144566/job/24294387285?pr=15105

@gmarull

**Pull request closed without merging**